location-awareness.md

```diff
@@ -89,3 +122,7 @@ In this case, PD will schedule different replicas of each datum to different dat
 - If the data zone cannot recover within a period of time, PD will remove the replica from this data zone.
 
 To sum up, PD maximizes the disaster recovery of the cluster according to the current topology. Therefore, if you want to reach a certain level of disaster recovery, deploy many machines in different sites according to the topology. The number of machines must be more than the number of `max-replicas`.
+
+However, with setting `isolation-level` to `zone`, PD will strictly ensure that each replica will isolated from each other at the zone level, even if guaranteeing this will not meet the number requirement of `max-replicas`. For example, there are three data zones z1/z2/z3 in the TiKV cluster. Under the requirement of 3 replicas, PD will dispatch the three replicas of the same Region to these three data zones. If there is a power outage in the z1 and it cannot be recovered after a period of time, PD will think that the region replica on z1 is no longer available, but because the `isolation-level` is set to `zone`, PD needs to strictly guarantee different region replicas will not fall on the same data zone. Because both z2 and z3 already have copies, PD will not perform any scheduling under the minimum isolation level limit of `isolation-level`, even if there are only two copies at this moment.
```

Comment:
```suggestion
However, if `isolation-level` is set to `zone`, PD will ensure that different replicas of a Region are isolated from each other at the zone level, even if guaranteeing this restriction does not meet the requirement of `max-replicas`. For example, a TiKV cluster is distributed across three data zones z1/z2/z3. Each Region has three replicas as required, and PD distributes the three replicas of the same Region to these three data zones respectively. If a power outage occurs in z1 and cannot be recovered after a period of time, PD determines that the Region replicas on z1 are no longer available. However, because `isolation-level` is set to `zone`, PD needs to strictly guarantee that different replicas of the same Region will not be scheduled on the same data zone. Because both z2 and z3 already have replicas, PD will not perform any scheduling under the minimum isolation level restriction of `isolation-level`, even if there are only two replicas at this moment.
```